src/Pages/kitchen_sink/KitchenSink.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react";
+import { CoralIsland, CrystalCaves, HomeImg } from "../../assets";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faArrowRight,
+  faArrowLeft,
+  faPlay,
+} from "@fortawesome/free-solid-svg-icons";
+
+const KitchenSink = () => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const prevSlide = () => {
+    const isFirstSlide = currentIndex === 0;
+    const newIndex = isFirstSlide ? data.length - 1 : currentIndex - 1;
+    setCurrentIndex(newIndex);
+  };
+  const nextSlide = () => {
+    const isLastSlide = currentIndex === data.length - 1;
+    const newIndex = isLastSlide ? 0 : currentIndex + 1;
+    setCurrentIndex(newIndex);
+  };
+
+  // Background images slide
+  const data = [
+    { img: HomeImg, label: "Home" },
+    { img: CoralIsland, label: "The Coral Island" },
+    { img: CrystalCaves, label: "The Crystal Caves" },
+  ];
+
+  interface ImgSlideStyle {
+    backgroundImage: string;
+    backgroundSize: string;
+    backgroundRepeat: string;
+    backgroundPosition: string;
+    width: string;
+    height: string;
```

Comment:
Optional / curiosity: Are you able to type these to the actual CSS property types?

Reply:
```
interface ImgSlideStyle {
  backgroundSize: BackgroundSize // "cover" "repeat" "contain"
}
```